src/checkers.cpp

```diff
@@ -51,5 +51,36 @@ namespace ada::checkers {
     }
     return accumulator;
   }
+
+  ada_really_inline constexpr bool check_domain(std::string_view input) noexcept {
+    if(input.size() > 255) {
```

Comment:
Why is it 255?

```
The length of the domain name, excluding the root label and its dot, is from 1 to 253.
```

Reply:
Yeah, that was my doubt. I was considering the 255 octets (bytes) as 255 chars. But I saw somewhere something about the last 2 being like "\0" or whatever other thing that means they arent "working" positions. I'll move to 253

Reply:
It is wrong.

Look at UCI:

https://github.com/unicode-org/icu/blob/bb0e745e25c99cc57055caf45c81b95ef63b25d4/icu4c/source/common/uts46.cpp#LL240C44-L240C76

If there is a dot in final position, then it we allow 254.